virt/virtual_machines/advanced_vm_management/virt-dedicated-resources-vm.adoc

```diff
@@ -4,16 +4,15 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-dedicated-resources-vm
 toc::[]
 
-Virtual machines can have resources of a node, such as CPU,
-dedicated to them in order to improve performance.
+Virtual machines can have resources of a node, such as CPU, dedicated to them in order to improve performance.
```

Comment:
And this also isn't related to your PR, but I think we should fix this phrasing.

Suggestion: "To improve performance, you can dedicate node resources, such as CPU, to a virtual machine."